src/crimson/mgr/client.cc

```diff
@@ -51,6 +51,21 @@ seastar::future<> Client::stop()
   co_await gates.close_all();
 }
 
+seastar::future<> Client::send(MessageURef msg)
+{
+  LOG_PREFIX(Client::start);
+  DEBUGDPP("{}", *this, *msg);
+  if (conn_lock.try_lock_shared()) {
+    if (conn) {
+      DEBUGDPP("sending {}", *this, *msg);
+      co_await conn->send(std::move(msg));
+    }
```

Comment:
I suppose the else case is possible only during initialization ?

Can we add a log to the else case ?